test/runtests.jl

```diff
@@ -109,6 +109,27 @@ end
         close(result)
         @test !isopen(result)
 
+        # the same but with tuple parameters
+        result = execute(
+            conn,
+            "SELECT typname FROM pg_type WHERE oid = \$1",
+            (16,);
```

Comment:
Make this something like `(1.0, 16)` so we can make sure no promotion is happening after the fact.

Reply:
Good idea.